src/managers/GuildBanManager.js

```diff
@@ -135,7 +135,7 @@ class GuildBanManager extends CachedManager {
    * @example
    * // Ban a user by id (or with a user/guild member object)
    * guild.bans.create('84484653687267328')
-   *   .then(user => console.log(`Banned ${user.username ?? user.id ?? user} from ${guild.name}`))
+   *   .then(user => console.log(`Banned ${user.user?.tag ?? user.tag ?? user} from ${guild.name}`))
```

Comment:
```suggestion
   *   .then(guildBan => console.log(`Banned ${guildBan.user?.tag ?? guildBan.user.id} from ${guildBan.guild.name}`))
```

Reply:
Although this function can resolve to either a GuildMember, a User or an ID, I think the name `user` is more appropriate than guildBan because that gives off the idea that it will return a `GuildBan` object. As for the rest, I did `user.user?.tag ?? user.tag ?? user` because it can resolve to a GuildMember (user.user.tag), a User (user.tag), or an id (user) and your approach would ignore the scenarios where User is returned

Reply:
Honestly, if it's that complicated, maybe just don't "show" such an example at all.

Reply:
or we could use instanceof checks to more explicitly say what we're checking for here

Reply:
The docs aren't a guide. Long and unnecessary examples defeat its purpose.

Reply:
Yeah ik but it should probably say the name of the person anyway.
Wouldn't it be better if it just always returned the user instead of trying to get a member?

Reply:
Changing the method signature and return values isn't really up for debate and I don't think a PR is up for debate either.

Reply:
Sure then. Should I just rename it to something like "banned"?

Reply:
Could also just name it `ban =>`

Reply:
Once again, that gives off the impression that it's a GuildBan and not a Member or a User imo

Reply:
You clearly show what it is afterwards though...? lol

Reply:
Also theres still the return value signature on the docs. If someone ignores all that, god help.

Reply:
Yeah I know but people can be really stupid, I think banned is better still

Reply:
I honestly think `banInfo` or something would be the best, but it might be too verbose.

And I don't really care about catering to what you refer to as "really stupid".

Reply:
I know it would just avoid unnecessary confusion
Either way, banned, ban or banInfo, you choose

Reply:
Just go with `banInfo` and `kickInfo` for now, we can always change that whenever, I don't want a single word be blocking the whole PR.